src/vs/workbench/contrib/notebook/browser/diff/notebookDiffList.ts

```diff
@@ -618,7 +621,9 @@ function buildSourceEditor(instantiationService: IInstantiationService, notebook
 		automaticLayout: false,
 		overflowWidgetsDomNode: notebookEditor.getOverflowContainerDomNode(),
 		readOnly: true,
-	}, {});
+	}, {
+		contributions: EditorExtensionsRegistry.getEditorContributions().filter(c => skipContributions.indexOf(c.id) === -1)
```

Comment:
Nit: the filter condition might be more readable with .includes() instead of .indexOf()